src/components/data_search/DatasetSearchTableDisplay.tsx

```diff
@@ -1,13 +1,17 @@
 import * as React from 'react';
 import { isEmpty, capitalize, isUndefined } from 'lodash';
-import {DatasetTerm} from 'src/types/model';
+import { DatasetTerm } from 'src/types/model';
 import SimpleTable from '../SimpleTable';
-import {Styles} from '../../libs/theme';
+import { Styles } from '../../libs/theme';
 import {
   type CellData,
   DatasetSearchTableTab,
 } from './DatasetSearchTableConstants';
-import {SnapshotSummaryModel} from '../../types/tdrModel';
+import { SnapshotSummaryModel } from '../../types/tdrModel';
```

Comment:
nit: I think linting in your IDE is doing this and might need a configuration update. Locally, for me, when I reformat the files, all of these white space changes are reverted.